docs/source/common/optimizers.rst

```diff
@@ -516,3 +516,49 @@ to perform a step, Lightning won't be able to support accelerators and precision
     ):
         optimizer = optimizer.optimizer
         optimizer.step(closure=optimizer_closure)
+
+-----
+
+Configure Gradient Clipping
+---------------------------
+To configure custom gradient clipping, consider overriding
+the :meth:`~pytorch_lightning.core.lightning.LightningModule.configure_gradient_clipping` function.
+By default it will get arguments :attr:`~pytorch_lightning.trainer.Trainer.gradient_clip_val` and
+:attr:`~pytorch_lightning.trainer.Trainer.gradient_clip_algorithm` from ``Trainer`` in their respective
```

Comment:
```suggestion
By default it will get arguments :attr:`~pytorch_lightning.trainer.trainer.Trainer.gradient_clip_val` and
:attr:`~pytorch_lightning.trainer.trainer.Trainer.gradient_clip_algorithm` from ``Trainer`` in their respective
```